.travis.yml

```diff
@@ -0,0 +1,11 @@
+script:
+  - if [[ "$TRAVIS_BRANCH" = "master" && "$TRAVIS_EVENT_TYPE" = "push" ]]; then
+      openssl aes-256-cbc -K $encrypted_a641d77e729e_key -iv $encrypted_a641d77e729e_iv
+        -in deploy/deploy_rsa.enc -out /tmp/deploy_rsa -d;
+
+      eval "$(ssh-agent -s)";
+      chmod 600 /tmp/deploy_rsa;
+      ssh-add /tmp/deploy_rsa;
+
+      ssh -o StrictHostKeyChecking=no root@$RUNNER_HOST "tsp git -C ./ruby-bench-docker pull --rebase origin master && tsp ./ruby-bench-docker/deploy/script";
```

Comment:
> better to build this on the web server

I am building images on bare metal server since that's where they are being used. We don't need docker images on web server.

The alternative way would be to build those on Travis and push them to docker hub.